tests/robottelo/test_decorators.py

```diff
@@ -36,20 +39,37 @@ def test_not_smoke(self):
 class BzBugIsOpenTestCase(TestCase):
     """Tests for :func:`robottelo.decorators.bz_bug_is_open`."""
     # (protected-access) pylint:disable=W0212
-    def setUp(self):  # noqa pylint:disable=C0103
+    # (test names are long to make it readable) pylint:disable=C0103
+    def setUp(self):
         """Back up objects and generate common values."""
         self.backup = decorators._get_bugzilla_bug
         self.bug_id = gen_integer()
-
-    def tearDown(self):  # noqa pylint:disable=C0103
+        self.upstream_backup = conf.properties.get('main.upstream')
```

Comment:
Agreed. I followed the already existing strategy in this test file. To update all the tests will be a separate effort and is documented in https://github.com/SatelliteQE/robottelo/issues/2780